rdrf/registry/patients/models.py

```diff
@@ -1118,6 +1123,11 @@ def family(self):
         return Family(self)
 
 
+class PatientStage(models.Model):
+    name = models.CharField(max_length=200, null=False)
+    previous = models.ManyToManyField('PatientStage', related_name='from_stage')
```

Comment:
Maybe use `'self'` instead of `'PatientStage'` to make it clear it is a self-reference.

The name/related name could be clearer? Maybe `allowed_prev_stages` and `allowed_next_stages`?

Reply:
Updated.